### DISCUSSION TITLE
About commit "repair changelogs"

### DISCUSSION THREAD

**Discussion by ehoogeveen-medweb** [csstools/postcss-plugins#1032] [Q&A] [answered]
Hi, I saw the recent [repair changelogs](https://github.com/csstools/postcss-plugins/commit/97a72168c6c984da117de2b665574472cf9167fa) commit, but I'm confused - did they need repairing?

The links seemed to render fine for me: [before change](https://github.com/csstools/postcss-plugins/blob/5166ce25379f65a7d715dd38b25ebedbbfd9b85f/experimental/postcss-gradient-stop-increments/CHANGELOG.md) | [after change](https://github.com/csstools/postcss-plugins/blob/97a72168c6c984da117de2b665574472cf9167fa/experimental/postcss-gradient-stop-increments/CHANGELOG.md).

**Comment by romainmenke** [comment, marked as answer]
When sharing the changelogs on other platforms, like Discord, the links needed to include the full url to github.

Otherwise they were perfect!

**Comment by ehoogeveen-medweb** [comment]
Aha, I see! I guess GitHub's cleverness with links doesn't extend to other platforms :)
